vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc

```diff
@@ -53,12 +53,19 @@ void RefillTokens::Refill(
 
   public_key_ = public_key;
 
+  nonce_ = "";
+
   RequestSignedTokens();
 }
 
 void RefillTokens::RetryGettingSignedTokens() {
   BLOG(INFO) << "Retry getting signed tokens";
 
+  if (nonce_.empty()) {
```

Comment:
Is there any danger of endlessly retriggering this?

Reply:
There is, however right now we have no retry count so I will raise a ticket

Reply:
OK, good idea.

Reply:
https://github.com/brave/brave-browser/issues/7226